# HG changeset patch
# User coffeys
# Date 1373968726 -3600
#      Tue Jul 16 10:58:46 2013 +0100
# Node ID 0b0490779c5b643b7a3e90ac6870c1ee4bd5a155
# Parent  2a023db33371ce5ee42134cf0d860ab9f0adff92
8017566: Backout 8000450 - Cannot access to com.sun.corba.se.impl.orb.ORBImpl
Reviewed-by: mchung, chegar

diff -r 2a023db33371 -r 0b0490779c5b src/share/lib/security/java.security-linux
--- jdk/src/share/lib/security/java.security-linux	Fri Oct 25 22:35:06 2013 +0100
+++ jdk/src/share/lib/security/java.security-linux	Tue Jul 16 10:58:46 2013 +0100
@@ -128,7 +128,6 @@
 # corresponding RuntimePermission ("accessClassInPackage."+package) has
 # been granted.
 package.access=sun.,\
-               com.sun.corba.se.impl.,\
                com.sun.xml.internal.,\
                com.sun.imageio.,\
                com.sun.istack.internal.,\
@@ -165,7 +164,6 @@
 # checkPackageDefinition.
 #
 package.definition=sun.,\
-                   com.sun.corba.se.impl.,\
                    com.sun.xml.internal.,\
                    com.sun.imageio.,\
                    com.sun.istack.internal.,\
diff -r 2a023db33371 -r 0b0490779c5b src/share/lib/security/java.security-solaris
--- jdk/src/share/lib/security/java.security-solaris	Fri Oct 25 22:35:06 2013 +0100
+++ jdk/src/share/lib/security/java.security-solaris	Tue Jul 16 10:58:46 2013 +0100
@@ -129,7 +129,6 @@
 # corresponding RuntimePermission ("accessClassInPackage."+package) has
 # been granted.
 package.access=sun.,\
-               com.sun.corba.se.impl.,\
                com.sun.xml.internal.,\
                com.sun.imageio.,\
                com.sun.istack.internal.,\
@@ -167,7 +166,6 @@
 # checkPackageDefinition.
 #
 package.definition=sun.,\
-                   com.sun.corba.se.impl.,\
                    com.sun.xml.internal.,\
                    com.sun.imageio.,\
                    com.sun.istack.internal.,\
diff -r 2a023db33371 -r 0b0490779c5b src/share/lib/security/java.security-windows
--- jdk/src/share/lib/security/java.security-windows	Fri Oct 25 22:35:06 2013 +0100
+++ jdk/src/share/lib/security/java.security-windows	Tue Jul 16 10:58:46 2013 +0100
@@ -129,7 +129,6 @@
 # corresponding RuntimePermission ("accessClassInPackage."+package) has
 # been granted.
 package.access=sun.,\
-               com.sun.corba.se.impl.,\
                com.sun.xml.internal.,\
                com.sun.imageio.,\
                com.sun.istack.internal.,\
@@ -168,7 +167,6 @@
 # checkPackageDefinition.
 #
 package.definition=sun.,\
-                   com.sun.corba.se.impl.,\
                    com.sun.xml.internal.,\
                    com.sun.imageio.,\
                    com.sun.istack.internal.,\
diff -r 2a023db33371 -r 0b0490779c5b test/java/lang/SecurityManager/CheckPackageAccess.java
--- jdk/test/java/lang/SecurityManager/CheckPackageAccess.java	Fri Oct 25 22:35:06 2013 +0100
+++ jdk/test/java/lang/SecurityManager/CheckPackageAccess.java	Tue Jul 16 10:58:46 2013 +0100
@@ -50,7 +50,6 @@
      */
     private static final String[] packages = {
         "sun.",
-        "com.sun.corba.se.impl.",
         "com.sun.xml.internal.",
         "com.sun.imageio.",
         "com.sun.istack.internal.",
